docker/Dockerfile.root5

```diff
@@ -28,12 +28,12 @@ RUN mkdir -p star-spack \
  && curl -sL https://github.com/spack/spack/archive/010e5761d6631eae102d1eec87e5e74a0e16ed82.tar.gz | tar -xz --strip-components 1 -C star-spack/spack
 
 RUN source star-spack/setup.sh \
+ && sed -e '/mesa/a\    variants: ~llvm' -i star-spack/environments/packages_gcc-4.8.5.yaml \
```

Comment:
Any reason to not modify the file directly in star-spack? A separate one can be created for containers if needed

Reply:
My goal was to contribute a thingy to build containers, so the focus here is to demonstrate its feasibility. We can make the change across repositories beforehand if you prefer.